tests/auto_class/test_auto_classifier.py

```diff
@@ -30,11 +30,26 @@ def test_create_classifier(indico_client, testdir_file_path):
         assert df.shape[1] == 3
         assert "class_a" and "class_b" in df["target"].tolist()
 
+
+def test_create_classifier_missing_files(indico_client, testdir_file_path):
+    aclass = AutoClassifier(
+        indico_client, os.path.join(testdir_file_path, "data/auto_class")
+    )
+    aclass.file_texts = [None, "a"]
+    aclass.file_classes = ["c", "d"]
+    aclass.file_paths = ["something", "something1"]
+    with tempfile.NamedTemporaryFile(suffix=".csv") as tf:
+        aclass.to_csv(tf.name)
+        df = pd.read_csv(tf.name)
+        assert df.shape[0] == 1
+        assert df.shape[1] == 3
+        assert "a" in df["text"].tolist()
+
+
 def test_create_classifier_too_few_classes(indico_client, testdir_file_path):
     aclass = AutoClassifier(
-        indico_client,
-        os.path.join(testdir_file_path, "data/auto_class/class_a")
-        )
+        indico_client, os.path.join(testdir_file_path, "data/auto_class/class_a")
```

Comment:
does this need to be black-ed?

Reply:
it got blacked (that's where the change on this came from)